src/components/ProjectContent.jsx

```diff
@@ -0,0 +1,60 @@
+import projectimg from '../assets/project.jpg';
+
+function ProjectContent() {

```

Comment:
This component should receive all the information that it displays as props.

Reply:
done